dev/release/verify-release-candidate.sh

```diff
@@ -531,16 +531,17 @@ test_integration() {
               $INTEGRATION_TEST_ARGS
 }
 
-clone_testing_repositories() {
-  # Clone testing repositories if not cloned already
-  if [ ! -d "arrow-testing" ]; then
-    git clone https://github.com/apache/arrow-testing.git
-  fi
-  if [ ! -d "parquet-testing" ]; then
-    git clone https://github.com/apache/parquet-testing.git
-  fi
-  export ARROW_TEST_DATA=$PWD/arrow-testing/data
-  export PARQUET_TEST_DATA=$PWD/parquet-testing/data
+clone_repositories() {
+  # Clone source and testing repositories if not cloned already
+  if [ ! -d "arrow" ]; then
+    git clone https://github.com/apache/arrow.git
```

Comment:
We should use the source archive (`apache-arrow-X.Y.Z.tar.gz`) not the master to align wheel version with test script version.

Reply:
ok

Reply:
@kou Updated ready for review